src/cargo.rs

```diff
@@ -132,12 +132,27 @@ impl CommandCargoExt for process::Command {
 pub(crate) fn cargo_bin_cmd<S: AsRef<str>>(name: S) -> Result<process::Command, CargoError> {
     let path = cargo_bin(name);
     if path.is_file() {
-        Ok(process::Command::new(path))
+        if let Some(runner) = cargo_runner() {
```

Comment:
In spirit of fully closing #139 (and to set user expectations), can you find an appropriate place to mention how cross support works?

Reply:
I've added some documentation on the `cargo_bin` documentation in c1d88ff85fe22042c270a6d54f189c2662c53e5c

There should be actually nothing special to do to make the tests run with `cross` — that's the whole point of this PR :-)